src/components/clusters/Clusters.tsx

```diff
@@ -38,15 +34,14 @@ const Clusters: React.FC<ClustersProps> = ({
   deleteCluster,
   clusterRows,
   clustersUIState,
-  clustersError,
 }) => {
   React.useEffect(() => {
     fetchClusters();
   }, [fetchClusters]);
 
   const errorState = (
     <PageSection variant={PageSectionVariants.light} isMain>
-      <ErrorState title={clustersError} fetchData={fetchClusters} />;
+      <ErrorState title="Failed to fetch clusters." fetchData={fetchClusters} />;
```

Comment:
Can we render text passed to the promise's reject function? It will be more flexible this way.